dotnet/samples/HomeAutomation/appsettings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "AzureOpenAiOptions": {
+    "Deployment": "",
+    "Endpoint": ""
+    // "ApiKey": "" /// Set this value in appsettings.Development.json, or using "dotnet user-secrets"
```

Comment:
Could we update this so that the user can use the same user-secrets as the KernelSyntaxExamples?

Reply:
Done.